db/migrations/2023011917330000-add-column-isReward-to-tokens-table.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const Tokens = require("../models/tokens.model");
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn("tokens", "isReward", {
+      type: Sequelize.BOOLEAN,
+      allowNull: true,
+      defaultValue: false,
+    });
+
+    Tokens.init(queryInterface.sequelize);
+//mesmo address mesmo column?
+    const tokens = await Tokens.findAll();
+    for (const token of tokens) {
+      console.log('token ->', token.isReward, token.isTransactional)
+      if (token.isTransactional === true) token.isReward = false;
+      else token.isReward = true;
+
```

Comment:
if `token` is reward and we changed the state, we should now have duplicates and these need to be removed

Reply:
@moshmage @MarcusviniciusLsantos i think that this migration instead of only set `true` or `false` to `isReward`, should iterate over tokens and check if the same token is allowed as transactional/reward, update `isTransactional` and `isReward` columns according to it and then remove the duplicated records.

Reply:
I adjust the migration to add in what it needs and delete the duplicate. I'll just need to adjust the rest to see if the logics are working correctly that way. because now IsAllowed is false when isTransactional and isReward is false